core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -548,6 +538,29 @@ class WorldScreen(
         }
     }
 
+    private fun displayTutorialTaskOnUpdate() {
+        tutorialTaskTable.clear()
+        val tutorialTask = getCurrentTutorialTask()
+        if (tutorialTask == "" || !game.settings.showTutorials || viewingCiv.isDefeated()) {
+            tutorialTaskTable.isVisible = false
```

Comment:
return here, then the rest of the function doesn't need to be in 'else'

Reply:
Previous function had only one level of nesting so this was less of a problem, but the new one has 2 levels. So we can flatten that out to just one.

Reply:
good point, fixed